### DISCUSSION TITLE
👏 欢迎来到 NoneBot 论坛

### DISCUSSION THREAD

**Discussion by stdrc** [nonebot/discussions#1] [通用]
你可以在这里：

- 问问题
- 解答别人的问题
- 分享新的 idea
- 分享你使用 NoneBot 的经验
- 与项目开发者们深入交流

**Comment by Darkerami** [comment]
请问用nonebot2内置的echo插件执行不了命令怎么办
![image](https://user-images.githubusercontent.com/118607168/217692419-ac7cd7db-18b3-4802-8e3e-d41a0df1b383.png)
![image](https://user-images.githubusercontent.com/118607168/217692463-3c11ac95-5295-4ec8-95d4-7d4ebc23a0d1.png)
